train.py

```diff
@@ -199,8 +200,9 @@ def train(tasks, model, args, device):
         # tasks.append(SemEval18Task())
         tasks.append(SemEval18SurpriseTask())
         tasks.append(SemEval18TrustTask())
-        tasks.append(SarcasmDetection())
+        # tasks.append(SarcasmDetection())
         tasks.append(OffensevalTask())
         for task in tasks:
-            model.add_task_classifier(task.NAME, task.get_classifier().to(device))
-    results = train(tasks, model, args, device)
+            model.add_task_classifier(task.get_name(), task.get_classifier())
```

Comment:
I think the .to(device) after get_classifier is required to avoid errors when model is on cuda